src/logging.h

```diff
@@ -27,16 +27,19 @@ static const char *const log_level_strings[] = {"DEBUG", "INFO", "WARNING",
 		sprintf(tx_buffer, "%" PRIu32 ",%s,%s:%d,%s," format "\n",     \
 			time, log_level_strings[level], __FILE__, __LINE__,    \
 			__func__, ##arg);                                      \
-		dma_write(tx_buffer, strlen(tx_buffer));                       \
+		serial_wait_send_available(10);                                \
+		serial_send(tx_buffer, strlen(tx_buffer));                     \
 	} while (0)
 
 #define LOG_DATA(format, arg...)                                               \
```

Comment:
Maybe two doc strings to explain the differences between LOG_DATA and LOG_MESSAGE and how to use them?

Reply:
b6250236957ec2f8652c1a42aee96e44753876ef

Reply:
great! :)